modules/selenium/build.gradle

```diff
@@ -1,16 +1,16 @@
 description = "Testcontainers :: Selenium"
 
 dependencies {
-    compile project(':testcontainers')
+    api project(':testcontainers')
 
     provided 'org.seleniumhq.selenium:selenium-remote-driver:3.141.59'
```

Comment:
BTW I've verified that the resulting POM files are effectively the same, using the approach described [here](https://github.com/testcontainers/testcontainers-java/pull/3347#discussion_r597968424), so I'm happy with this.